### DISCUSSION TITLE
Course Editor: how to define travel in field?

### DISCUSSION THREAD

**Discussion by Nad42** [Courseplay/courseplay#7410] [Q&A]
Hi, just wondering how to define waypoints where you want it to lift the tool drive somewhere/around something before continuing?

Is there some technique of using turns?

Thanks!

**Comment by Tensuko** [comment]
There is a waypoint called start turnd and end turn. The rest is done by the turn or pathfinder path if useing this in turn.
What do you try to do ? Maybe I can give you a better solution.

**Reply by Nad42** [reply]
Yeah the start and end turn work great in most instances but I've got a few places where between the start and end there is a building or tree and it just crashes straight through them... it would be nice if you could put a path between the start and end to make it go where you want/avoid certain things whilst its turning.

**Comment by Tensuko** [comment]
There is no way to put a path between them.
When you have a Field that is L oder U shaped, just tell the driver to use Pathfinder in turns:
![pathfinder turns](https://user-images.githubusercontent.com/27344744/128606290-bf27f2eb-74d8-4d13-a650-b915a64c3096.png)
It wil drive along the headland untill the turn end point and goes back into its line.

As I said... rather tell us what you want to do and what is your problem, then try to find any special solution...

**Reply by Nad42** [reply]
Firstly; thankyou very much for the help, its appreciated!
now I have tried turning pathfinding on a couple of times but it always ends up doing some really weird stuff, I've even watched it still drive straight into a tree with it on... the main issue I have with it is that it is a lot slower as it then insists on driving round the edge of the field instead of taking a more direct path when it is available... and it always uses the outer most headland path which is ridiculous when sometime I have 5 headlands on so it dives miles just to go out to the outside then back again...

**Reply by Tensuko** [reply]
Well I know what you talk about, I sometimes have the same Issue, but I hope that we can get it solved with the new pathfinder... We will see what we can do.
Beside that, there is no other option...
Well maybe create the mid lines in a different direction.
The Course Generator is able to split the field in 'parts' and when he does that, there will be travel lanes created.
Maybe you can get something like that:
![ushape](https://user-images.githubusercontent.com/27344744/128673281-2c21dbad-4faa-408d-99df-0b7c6e11bfaf.png)


You could look up in the course file if you find information about the difference, but I think that is nothing to edit.

**Reply by Nad42** [reply]
How do you split it into parts with the course editor?

Nice drawing BTW :P

**Reply by Tensuko** [reply]
The Generator will do it automatically in shapes like that.
If you know the Map 'Irgendwo in Thüringen V2' there is a Field 28.
The Generator will create some of those up/down parts.

**Reply by Nad42** [reply]
But how do you manually create your own if the generator doesn't create them properly?

**Reply by Tensuko** [reply]
Can't, play arround with Generator options..
